tests/by-util/test_cp.rs

```diff
@@ -517,12 +517,12 @@ fn test_cp_arg_interactive_update() {
 
 #[test]
 #[cfg(not(any(target_os = "android", target_os = "freebsd")))]
-#[ignore = "known issue #6019"]
-fn test_cp_arg_interactive_update_newer() {
+fn test_cp_arg_interactive_update_overwrite_older() {
     // -u -i *WILL* show the prompt to validate the override.
     // Therefore, the error code depends on the prompt response.
     let (at, mut ucmd) = at_and_ucmd!();
     at.touch("b");
+    std::thread::sleep(Duration::from_secs(1));
```

Comment:
could you please add a comment why sleep ?
and can we make it shorter ?
thanks

Reply:
yes, this issue is what I posted in the chat. The test itself was failing initially. The reason is that the two create calls are so close to each other that the files get at the same time. To mitigate that we have two options. 1) add a delay and 2) set time explicitly.
I tried to set time explicitly initially,  but the libraries I used were unstable (FileTimes). Please let me know if you have some option to set time while creating the file. I had no other option than to use 1.
Sorry, I did not get what you meant by shorter...

Reply:
For 2) you could use something like:
```
let _ = at.make_file("b").set_modified(SystemTime::UNIX_EPOCH);
```

Reply:
> For 2) you could use something like:
> 
> ```rust
> let _ = at.make_file("b").set_modified(std::time::SystemTime::UNIX_EPOCH);
> ```
> 
> Update: Please ignore it, I just realized this snippet requires Rust 1.75 :|

yes, I tried FileTimes but it did not work as you said...